randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -855,8 +865,10 @@ Extra - room_id: [20]
       Level 4 Locks and Speed Booster and Jump Extends (Intermediate) and Shinespark Tricks (Beginner) and Disabled Door Lock Rando and Disabled Entrance Rando
   > Door to Neglected Save Room
       Any of the following:
-          Space Jump or Wall Jump (Beginner)
+          Space Jump
           Stand On Frozen Enemies (Intermediate) and Can Freeze Enemies With Any Weapon
+          # Break the blocks in a certain way and Wall Jump: https://youtu.be/cB_xiz5mbr8
```

Comment:
the middle->top part from this video wasn't actualy in logic o.o
Can you add it please?

Reply:
When I was looking at this room I was thinking of possibly re-doing the logic by adding a "center of room" node instead of using the Save Room door.

Reply:
that's not needed, using the save door is perfectly fine being used for a middle-level node.

Reply:
Do you want the trick to show the same video? I can add a timestamp in the link or leave it as is.

Reply:
you can leave the link as is